src/plugins/htmlAudioPlayer/plugin.js

```diff
@@ -124,13 +124,13 @@ class HtmlAudioPlayer {
                 }
 
                 if (normalizationGain) {
-                    self.gainNode.gain.value = Math.pow(10, normalizationGain / 20);
-                    self.normalizationGain = self.gainNode.gain.value;
+                    self.normalizationGain = Math.pow(10, normalizationGain / 20);
+                    self.gainNode.gain.value = browser.safari ? elem.volume * self.normalizationGain : self.normalizationGain;
```

Comment:
You can probably extract volume multiplication, placing following after else block:
```js
if (browser.safari) {
    // Gain value is absolute in Safari. Add volume from the slider
    self.gainNode.gain.value *= elem.volume;
}
```
_Comment can be enhanced._